validphys2/src/validphys/photon/compute.py

```diff
@@ -247,3 +262,39 @@ def generate_errors(self, replica):
         u, s, _ = np.linalg.svd(self.error_matrix, full_matrices=False)
         errors = u @ (s * rng.normal(size=7))
         return errors
+    
+    def load_photon(self):
+      """Load the photon resource using the Loader class."""
+      path_to_photon = loader.check_photonQED(self.theoryid.id, self.luxpdfset._name)
+      log.info(f"Loading photon QED set from {path_to_photon}")
+
+      interpolator = []
+      integral = []
+
+      # Load the needed replicas
+      for replica in self.replicas:
+          # As input replica for the photon computation we take the MOD of the luxset_members to
+          # avoid failing due to limited number of replicas in the luxset
+          photonreplica = (replica % self.luxpdfset_members) or self.luxpdfset_members
+
+          photon_array = np.load(path_to_photon / f"replica_{photonreplica}.npz")["photon_array"]
+          interpolator.append(interp1d(XGRID, photon_array, fill_value="extrapolate", kind="cubic"))
+          integral.append(trapezoid(photon_array, XGRID))
+      
+      self.interpolator = interpolator
+      self.integral = np.stack(integral, axis=-1)
+      return
+    
+def compute_photon(theoryid, fiatlux):
+    """Function to compute the photon PDF set.""" 
+    luxset = fiatlux['luxset'].load()
+    force_compute = fiatlux.get('compute_in_setupfit', False)
+    replicas = list(range(1, luxset.n_members))
+
+    # Return None and avoid pickling issues with the phoyon class.
+    def wrapper_fn(replica, theoryid, fiatlux, force_compute):
+        Photon(theoryid, fiatlux, replicas=[replica], save_to_disk=True, force_computation=force_compute)
```

Comment:
```suggestion
        Photon(theoryid, fiatlux, replicas=[replica], save_to_disk=force_compute, force_computation=force_compute)
```

I think _for the time being_ it would be better to only save it to disk if you have the `compute_in_setupfit` flag on.

Also, let's call this function something like `compute_photon_to_disk` or something like that, since it doesn't actually return a photon! (you might want to return the list of files with the photon or something)